nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -224,6 +224,10 @@ PLUGINS = []
 REMOTE_AUTH_AUTO_CREATE_USER = False
 REMOTE_AUTH_HEADER = "HTTP_REMOTE_USER"
 
+# Set the SESSION_ENGINE
+# By default, Nautobot will use 'django.contrib.sessions.backends.db' however if you need a maintenance mode / read-only instance then this can be overriden
+SESSION_ENGINE = os.getenv("NAUTOBOT_SESSION_ENGINE", "django.contrib.sessions.backends.db")
+
```

Comment:
Can this be implemented in `nautobot/core/settings.py` instead? (See also #2578)